src/Spago/Glob.purs

```diff
@@ -49,7 +49,7 @@ gitignoreGlob base =
       ( \line -> do
           let
             resolve a = Path.concat [ base, a ]
-            pat a = resolve $ unpackPattern a
+            pat a = withForwardSlashes $ resolve $ unpackPattern a
```

Comment:
:melting_face:
`Path.concat ["foo\\bar", "**"] == "foo\\bar\\**"`
`\**` and `/**` are ***very*** different.